### ISSUE TITLE
Error when publishing service provider

### ISSUE THREAD

**Issue by Naoray** [faustbrian/Laravel-Voteable#9]
- Version: 2.1
- PHP Version: 7.1

## Expected behavior:
Service Provider is published.

## Actual behavior:
Error appears claiming "Class 'BrianFaust\ServiceProvider\ServiceProvider' not found"
![image](https://cloud.githubusercontent.com/assets/10154100/22397066/39512adc-e569-11e6-9fca-a9760774a928.png)

## Steps to reproduce the behavior:
Add Package via composer, run `php artisan vendor:publish --provider="BrianFaust\Voteable\VoteableServiceProvider"`

## Proposed solution:
Rename Reference to ServiceProvider:
```
use BrianFaust\ServiceProvider\AbstractServiceProvider as ServiceProvider;
```

**Comment by Naoray** [comment]
=> #10

**Comment by faustbrian** [comment]
Merged and `2.1.1` tagged.
